### PR TITLE
bluetooth.py: migrate to ravel

### PR DESCRIPTION
**Pull request by thoradia**
I submit this pull request to indicate that I am working on bluetooth settings
The changes do not break the settings add-on
They unfortunately do not fix it either
Work in progress, thus

### CODE REVIEW

**Comment by thoradia** [comment]
This re-enables basic functions such as connecting/pairing/disconnecting
Please test and merge

File: resources/lib/dbus_bluez.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import dbus_utils
```
Comment (chewitt) [inline resources/lib/dbus_bluez.py:1]: copyright header

File: resources/lib/modules/bluetooth.py [hunk 2]
```diff
@@ -73,48 +76,26 @@ def exit(self):
...
     @log.log_function()
-    def adapter_info(self, adapter, name):
-        adapter_interface = dbus.Interface(oe.dbusSystemBus.get_object('org.bluez', adapter.object_path),
-                                           'org.freedesktop.DBus.Properties')
-        res = adapter_interface.Get('org.bluez.Adapter1', name)
-        adapter_interface = None
-        oe.dbg_log('bluetooth::adapter_info', 'exit_function', oe.LOGDEBUG)
-        return res
+    def init_adapter(self):
+        dbus_bluez.adapter_set_alias(self.dbusBluezAdapter, 'LibreELEC')
```
Comment (chewitt) [inline resources/lib/modules/bluetooth.py:86]: hardcoded LibreELEC should be replaced with @@DISTRONAME@@ or something we can sed at build time to support downstream..

**Comment by thoradia** [comment]
Thank you for the feedback @chewitt 
Bluetooth now advertises the hostname or the distribution name
Headers have been added
I also refactored hostname/regdomain logic and fixed connman error handling
This can be merged

**Review by chewitt — APPROVED** [review]

**Pull request merged**